models/news.models.js

```diff
@@ -49,3 +48,20 @@ exports.selectArticleById = (id) => {
       return rows[0];
     });
 };
+
+exports.selectCommentsByArticleId = (id) => {
+  return db
+    .query(
+      `SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at ASC`,
+      [id]
+    )
+    .then((result) => {
+      if (result.rowCount === 0) {
```

Comment:
Have you checked the reason for the query returning empty rows? Is it that the article doesn't exist? What about if the article did exist but it had no comments? 🤔